configs/simswaphq_gpen_512.yaml

```diff
@@ -0,0 +1,7 @@
+---
+swap_type: simswap
+parsing_model_path: ./saved_models/simswap/parsing_model/checkpoint/79999_iter.pth
+arcface_model_path: ./saved_models/simswap/arcface_model/arcface_checkpoint.tar
+checkpoints_dir: ./saved_models/simswap/checkpoints
+crop_size: 512
+gpen_type: gpen_512
```

Comment:
`gpen` can also be used with `simswap` (the 224 model) and with `faceswap_cv2`. I don't think that for every possible combination we should create a config file. We can just add this to the command used `--gpen_type <gpen_type>`

Reply:
i see. in that case, I will update the README as well.

Reply:
[cf93e9f](https://github.com/sensity-ai/dot/pull/17/commits/cf93e9f6aa2f3445fdb885e231e254720fe9b2cc)